functions.php

```diff
@@ -14,6 +12,11 @@ function ywca_enqueue_styles() {
 add_action( 'wp_enqueue_scripts', 'bootstrap_enqueue_styles' );
 add_action( 'wp_enqueue_scripts', 'ywca_enqueue_styles' );
 
+
+
+add_action( 'wp_enqueue_scripts', 'bootstrapstarter_enqueue_styles' );
+add_action( 'wp_enqueue_scripts', 'bootstrapstarter_enqueue_scripts' );
+
```

Comment:
Bootstrapstarter functions don't exist, use the enqueue functions from functions.php in master.